docs/how-to/troubleshoot_user_registration_issues.md

```diff
@@ -0,0 +1,98 @@
+
+## 1. Subscriber registration failed with `REGISTRATION-REJECT-EVENT` error
+
+### Symptoms
+The `juju debug-log sdcore-gnbsim-k8s/leader` command reports an error in the UE registration:
+```console
+handling event: REGISTRATION-REJECT-EVENT	{"component": "GNBSIM", "category": "SimUe", "supi": "imsi-001010100007487"}
+```
+
+### Recommended Actions
+This error is typically raised when the UE is not provisioned in SD-Core or the link between AMF and AUSF or between AUSF and UDM/UDR is not established.
+
+#### Provisioning
+Validate whether the UE is provisioned in SD-Core.
+
+#### Configuration
+Validate the AMF is able to select an AUSF by inspecting logs using `microk8s.kubectl logs -n sdcore amf-0 -c amf -f`. The logs should report:
+```console
+INFO	gmm/handler.go:1509	Authentication procedure	{"component": "AMF", "category": "GMM", "amf_ue_ngap_id": "AMF_UE_NGAP_ID:36", "suci": "suci-0-001-01-0-0-0-0100007487"}
+INFO	message/send.go:80	send Authentication Request	{"component": "AMF", "category": "GMM", "amf_ue_ngap_id": "AMF_UE_NGAP_ID:36", "suci": "suci-0-001-01-0-0-0-0100007487"}
+INFO	gmm/handler.go:583	Handle InitialRegistration	{"component": "AMF", "category": "GMM", "amf_ue_ngap_id": "AMF_UE_NGAP_ID:2", "suci": "suci-0-001-01-0-0-0-0100007487", "supi": "SUPI:imsi-001010100007487"}
+INFO	gmm/handler.go:2254	Handle Registration Complete	{"component": "AMF", "category": "GMM", "amf_ue_ngap_id": "AMF_UE_NGAP_ID:2", "suci": "suci-0-001-01-0-0-0-0100007487", "supi": "SUPI:imsi-001010100007487"}
+INFO	fsm/fsm.go:99	handle event[ContextSetup Success], transition from [ContextSetup] to [Registered]	{"component": "LIB", "category": "FSM"}
+```
+
+Validate the AUSF is able to select an UDM by inspecting logs using `microk8s.kubectl logs -n sdcore ausf-0 -c ausf -f`. The logs should report:
+```console
+INFO	producer/ue_authentication.go:84	HandleUeAuthPostRequest	{"component": "AUSF", "category": "UeAuthPost"}
+INFO	producer/ue_authentication.go:66	Auth5gAkaComfirmRequest	{"component": "AUSF", "category": "5gAkaAuth"}
+```
+
+Validate the UDM is able to select an UDR by inspecting logs using `microk8s.kubectl logs -n sdcore udm-0 -c udm -f`. The logs should report no errors after this line:
+```console
+INFO	producer/generate_auth_data.go:84	handle GenerateAuthDataRequest	{"component": "UDM", "category": "UEAU"}
+```
+
+## 2. Subscriber authentication failed with `AUTHENTICATION-REJECT-EVENT` error
+
+### Symptoms
+The `juju debug-log sdcore-gnbsim-k8s/leader` command reports an error in the UE authentication:
+```console
+handling event: AUTHENTICATION-REJECT-EVENT	{"component": "GNBSIM", "category": "SimUe", "supi": "imsi-001010100007487"}
+```
+
+### Recommended Actions
+This error is typically raised when the UE is provisioned in SD-Core but the authentication (OPc and/or key) data in SD-Core does not match the actual UE data or the link between AMF and AUSF or between AUSF and UDM/UDR is not established.
```

Comment:
This sentence is not a "Recommended Action" but an explanation, if those are needed, please add a section for it.